scylla/src/cluster/node.rs

```diff
@@ -78,6 +80,9 @@ pub struct Node {
 
     // If the node is filtered out by the host filter, this will be None
     pool: Option<NodeConnectionPool>,
+
+    #[cfg(test)]
+    enabled_as_connected: AtomicBool,
```

Comment:
:wrench: Please take the explanation written in the commit message and put in a docstring here.